tests/benchmarks/current_consumption/system_off/testcase.yaml

```diff
@@ -0,0 +1,17 @@
+common:
+  sysbuild: true
+  tags: ci_tests_benchmarks_current_consumption ppk_power_measure
+
+tests:
+  benchmarks.current_consumption.systemoff.gpio_wakeup:
+    platform_allow:
+      - nrf54l15dk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15dk/nrf54l15/cpuapp
+    extra_args: DTC_OVERLAY_FILE="boards/nrf54l15dk_nrf54l15_cpuapp_gpio_wakeup.overlay"
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - ../test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption_with_wakeups"
```

Comment:
The path should be like: ${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py:...